code/mathematical-algorithms/replace_0_with_5/change0to5.py

```diff
@@ -14,3 +16,11 @@
 
 except Exception:
   print("You did not specify an integer")
+
+  
+# python 3 code:
```

Comment:
You could use `sys.version_info` to check in what version the code is running and then run only the appropriate code for the version.

Reply:
Done